woocommerce/woocommerce-gateway-stripe
# Update express checkout section UI

Fixes #2393

## Changes proposed in this Pull Request:
This PR updates the Express Checkout section in the Stripe payment method settings.

Context: p1658843212415559-slack-CHG7MTCAF

<img width="1046" alt="Screenshot 2022-10-26 at 17 34 42" src="https://user-images.githubusercontent.com/8667118/198063388-96a4921e-8c07-4799-91dd-2b148dffe7d4.png">

<img width="1051" alt="Screenshot 2022-10-26 at 17 43 48" src="https://user-images.githubusercontent.com/8667118/198063414-59dee435-0ec4-495c-89bc-0f7372ae20b9.png">


## Testing instructions
- Navigate to `WooCommerce` -> `Settings` -> `Payments` -> `Stripe`.
- Verify the Apple & Google Pay and Stripe Link are displayed in a separate section, `Express Checkouts`.
- Verify Stripe Link is available only when the Credit / Debit Card is activated.
- Verify enabling/disabling payment methods are fully functional on both the back-end and front-end.
- Navigate to `Apple & Google Pay` -> `Customize`.
- Verify the `Enable Apple Pay / Google Pay` is displayed in a separate section.
- Verify all the settings are saved and fully functional on both the back-end and front-end.

---


**Post merge**
